app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
@noslouch the `__` is to define an Element that belongs to a Block. And the `--` is to define a Modifier (to an element or a block). 

So for instance: `.alert__header--success`, or `.alert--success`.